src/main/java/in/projecteka/consentmanager/consent/ConsentArtefactRepository.java

```diff
@@ -194,6 +213,19 @@
         }));
     }
 
+    public Mono<Void> updateConsentArtefactStatus(String consentId, ConsentStatus status) {
+        return Mono.create(monoSink -> dbClient.preparedQuery(UPDATE_CONSENT_ARTEFACT_STATUS_QUERY)
+                .execute(Tuple.of(status.toString(),
+                        LocalDateTime.now(),
+                        consentId),
+                        updateHandler -> {
+                            if (updateHandler.failed()) {
+                                monoSink.error(new Exception("Failed to update consent artefact status"));
```

Comment:
you have to return after this statement.

Reply:
Done

Reply:
done

Reply:
done